packages/web/src/components/notification/Notifications/TipSentNotification.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+import { TipSent } from 'common/store/notifications/types'
+
+import { AudioText } from './AudioText'
+import { NotificationBody } from './NotificationBody'
+import { NotificationFooter } from './NotificationFooter'
+import { NotificationHeader } from './NotificationHeader'
+import { NotificationTile } from './NotificationTile'
+import { NotificationTitle } from './NotificationTitle'
+import { ProfilePicture } from './ProfilePicture'
+import styles from './TipSentNotification.module.css'
+import { TwitterShareButton } from './TwitterShareButton'
+import { UserNameLink } from './UserNameLink'
+import { IconTip } from './icons'
+
+const messages = {
+  title: 'Your Tip Was Sent!',
+  sent: 'You successfully sent a tip of',
+  audio: '$AUDIO',
+  to: 'to'
+}
+
+type TipSentNotificationProps = {
+  notification: TipSent
+}
+
+export const TipSentNotification = (props: TipSentNotificationProps) => {
+  const { notification } = props
+  const { user, value, timeLabel, isRead } = notification
+
+  return (
+    <NotificationTile notification={notification}>
+      <NotificationHeader icon={<IconTip />}>
+        <NotificationTitle>{messages.title}</NotificationTitle>
+      </NotificationHeader>
+      <NotificationBody className={styles.body}>
+        <ProfilePicture className={styles.profilePicture} user={user} />
+        <span>
+          {messages.sent} <AudioText value={value} /> {messages.to}{' '}
+          <UserNameLink user={user} notification={notification} addMargin />
```

Comment:
Not a big fan of `addMargin` - it also looks like it's not necessary anymore. Now that the `ArtistPopover` is a span, it should keep the whitespace?

Think maybe we can remove this?

Reply:
yes down for sure, ahh that is why the whitespace was messed up, will do